Smart-Contract-Verifier-Server/src/main/kotlin/com/savannasolutions/SmartContractVerifierServer/models/Agreements.kt

```diff
@@ -9,13 +9,13 @@ import javax.persistence.*
 @Entity
 @Table(name = "Agreements")
 data class Agreements(@Id @GeneratedValue val ContractID:UUID,
-                      var blockchainID: String?,
+                      var blockchainID: String? = "",
```

Comment:
Could cause confusion between empty string and null if both are allowed

Reply:
I have updated blockchainID accordingly. Please recheck